### PR TITLE
@orta => remove categories section from fair guide

### PR DESCRIPTION
**Pull request by 1aurabrown**
fixes https://github.com/artsy/eigen/issues/190

### CODE REVIEW

**Comment by orta** [comment]
kill genes

**Comment by orta** [comment]
Giphy

**Comment by orta** [comment]
Giphy

File: Artsy Tests/ARBrowseFeaturedLinksCollectionViewTests.m [hunk 1]
```diff
@@ -101,42 +101,6 @@ - (NSString *)reuseIdentifier;
...
-            viewController.view = collectionView;
-            [viewController ar_presentWithFrame:CGRectMake(0, 0, 768, 1024)];
-            expect(collectionView).will.haveValidSnapshot();
-        });
-
-        itBehavesLike(@"general view setup", nil);
-    });
 });
 
 describe(@"setupPaging", ^{
```
Comment (1aurabrown) [inline Artsy Tests/ARBrowseFeaturedLinksCollectionViewTests.m]: @orta i just realized i should have deleted this whole describe block. i thought i had done this. will do in a separate pr.

Comment (orta) [inline Artsy Tests/ARBrowseFeaturedLinksCollectionViewTests.m]: :+1:
